docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,97 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+This section will go over how to upgrade Alluxio in your Kubernetes cluster with `kubectl`.
+
+**Step 1: Upgrade the docker image version tag**
+ 
+Each released Alluxio version will have the corresponding docker image released on [dockerhub](https://hub.docker.com/r/alluxio/alluxio).
+
+You should update the `image` field of all the Alluxio containers to use the target version tag. Tag `latest` will point to the latest stable version.
+It is recommended that Alluxio masters and workers are running on the same version for the best compatibility.
+
+For example, if you want to upgrade Alluxio to the latest stable version, update the containers as below:
+
+```yaml
+containers:
+- name: alluxio-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+- name: alluxio-job-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+```
+
+**Step 2: Stop running Alluxio master and worker Pods**
+
+Kill all running Alluxio worker Pods.
+
+```console
+$ kubectl delete -f alluxio-worker.yaml
+```
+
+Then kill all running Alluxio master Pods by killing each StatefulSet and each Service.
+
+```console
+$ let masterCount = {number of your masters minus 1}
```

Comment:
Jesus how can I do the range exclusive of the upper bound...So I can get rid of this `minus 1` here.

Reply:
@madanadit The for loop has been replaced by using label selector `kubectl delete statefulset -l app=alluxio`. This is a cleaner solution (although somehow relies on masters being StatefulSet). We can discuss on better labeling.

The extra YAML changes are to apply the labels for doing this.